### ISSUE TITLE
Submitting Images in React Final Form

### ISSUE THREAD

**Issue by HarshArora-1205** [final-form/react-final-form#1036]
### Are you submitting a **bug report** or a **feature request**?

I am submitting a feature request. There is no documentation related to Images in React-Final-Form Docs. Further there is no tutorial or not much stack overflow support available.
Now it's being difficult to tweak the form as per my need and it will be a hard change in my project to switch to another form. It would be helpful if documentation is updated or if new feature related to that comes up (if not existing yet).

### What is the current behavior?

Currently I can't get behind the scenes info for controlled input for uploading multiple files. Moreover after trying modifying onChange() handler, using states to set data, the form automatically re-renders which doesn't help in any way.

### What is the expected behavior?

If there could be documentation related to handling images and uploading multiple images and passing it to backend, it will be very helpful. Further providing methods how to tweak onChange() or other properties in documentaion, will also be helpful.

### What's your environment?

I am currently using Fedora 38, React version: 18.2.0, React Final Form version: 6.5.9, Final Form Version: 4.20.9, Browser: Mozilla Firefox/ Brave.

### Other information

In summary, I wish to upload multiple images in react final form, for which I tried to pass encType='mutlipart/form-data' in form itself as well as req header, tried to set files state right
after getting input (by tweaking onChange() function). Still proper Documentation is required as there is not much online support available.
